tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -1623,6 +1623,49 @@ def test_issue_in_search_and_columns(self):
                 else:
                     assert data[0].get("issue", None) is None
 
+    def test_issue_negation(self):
+        self.login_as(user=self.user)
+
+        project1 = self.create_project()
+        project2 = self.create_project()
+        event1 = self.store_event(
+            data={
+                "event_id": "a" * 32,
+                "transaction": "/example",
+                "message": "how to make fast",
+                "timestamp": self.two_min_ago,
+                "fingerprint": ["group_1"],
+            },
+            project_id=project1.id,
+        )
+        self.store_event(
+            data={
+                "event_id": "b" * 32,
+                "transaction": "/example",
+                "message": "how to make fast",
+                "timestamp": self.two_min_ago,
+                "fingerprint": ["group_2"],
+            },
+            project_id=project2.id,
+        )
+
+        with self.feature(
+            {"organizations:discover-basic": True, "organizations:global-views": True}
+        ):
+            response = self.client.get(
+                self.url,
+                format="json",
+                data={
+                    "field": ["title"],
+                    "query": "!issue:{}".format(event1.group.qualified_short_id),
+                },
+            )
+
+            assert response.status_code == 200, response.content
+            data = response.data["data"]
+            assert len(data) == 1
+            assert data[0]["title"] == event1.title
```

Comment:
@markstory this should be the title of the 2nd event.

Reply:
Whoops.

Reply:
This test was passing by luck as the two events have the same title :blush: I'll get that fixed.